mix/CodeModel.cpp

```diff
@@ -358,7 +360,7 @@ void CodeModel::gasEstimation(solidity::CompilerStack const& _cs)
 {
 	if (m_gasCostsMaps)
 		m_gasCostsMaps->deleteLater();
-	m_gasCostsMaps = new GasMapWrapper(this);
+	m_gasCostsMaps = new GasMapWrapper();
```

Comment:
1. A raw `new` allocation? Does it mean manual memory management?
2. Avoid `()` after object creation expression. You can use `{}` or nothing. http://stackoverflow.com/questions/1424510/most-vexing-parse-why-doesnt-a-a-work